.github/workflows/update-brew-formulae.yml

```diff
@@ -0,0 +1,16 @@
+name: "Update homebrew tap"
+on:
+  release:
+    types: [published]
+  schedule:
```

Comment:
What is the reason for the cron job? Why not just on release: published?

Reply:
From reading around, it's not always triggered, so it's used to be sure.